### PR TITLE
[WIP] add consolelink for the routes

### PR DESCRIPTION
**Pull request by myeung18**
Signed-off-by: Marco Yeung <myeung@redhat.com>

### CODE REVIEW

File: apicurito/deploy/cluster_role.yaml [hunk 1]
```diff
@@ -7,8 +7,10 @@ rules:
   - console.openshift.io
   resources:
   - consoleyamlsamples
+  - consolelinks
   verbs:
   - get
   - create
+  - list
   - update
   - delete
```
Comment (redhatHameed) [inline apicurito/deploy/cluster_role.yaml:16]: @myeung18 for this operator CSV and _role.yaml generated from go code, so you need to add this in go code.

File: apicurito/deploy/cluster_role_binding.yaml [hunk 2]
```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
   - kind: ServiceAccount
     name: apicurito
-    namespace: placeholder
+    namespace: ms-test
```
Comment (redhatHameed) [inline apicurito/deploy/cluster_role_binding.yaml:12]: make sure, revert this change

File: apicurito/deploy/manifests/0.2.0/apicuritooperator.v0.2.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -112,7 +112,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: apicurito-operator
-                image: registry.redhat.io/fuse7-tech-preview/fuse-apicurito-operator:1.7
+                image: quay.io/myeung/apicurito-operator:v0.2
```
Comment (redhatHameed) [inline apicurito/deploy/manifests/0.2.0/apicuritooperator.v0.2.0.clusterserviceversion.yaml:115]: make sure, revert this change

File: apicurito/deploy/operator.yaml [hunk 4]
```diff
@@ -33,7 +33,7 @@ spec:
               fieldPath: metadata.name
         - name: OPERATOR_NAME
           value: apicurito-operator
-        image: registry.redhat.io/fuse7-tech-preview/fuse-apicurito-operator:1.7
+        image: quay.io/myeung/apicurito-operator:v0.2
```
Comment (redhatHameed) [inline apicurito/deploy/operator.yaml:36]: same here, revert this change once you done your testing

File: apicurito/deploy/role.yaml [hunk 5]
```diff
@@ -13,6 +13,7 @@ rules:
   - events
   - configmaps
   - secrets
+  - serviceaccounts
   verbs:
   - '*'
 - apiGroups:
```
Comment (redhatHameed) [inline apicurito/deploy/role.yaml:19]: need to be added in go code for generating this file

File: apicurito/scripts/go-build.sh [hunk 6]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-REGISTRY=quay.io/${USER}
+REGISTRY=quay.io/myeung
```
Comment (redhatHameed) [inline apicurito/scripts/go-build.sh:2]: make sure, revert this change

File: apicurito/deploy/cluster_role_binding.yaml (hunk 2 above)
Comment (myeung18) [reply apicurito/deploy/cluster_role_binding.yaml:12]: dont need to review just yet. 
things are in WIP.
